generators/create-react-app/index.test.ts

```diff
@@ -15,7 +15,6 @@ describe('When running the generator', () => {
 
         await helpers.run(__dirname)
             .cd(root)
-            .withOptions({ skipInstall: false })
```

Comment:
pourquoi on en a plus besoin finalement ?

Reply:
Because without it `yarnInstall` steps where skipped, since we now run `yarn` with `spawnCommand` it is no longer necessary.